ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```diff
@@ -1,103 +1,46 @@
-ptp "Some long running PTP test for example"
-
-EOF
-}
-
-
-function create_tests_temp_skip_list_11 {
-# List of temporarly skipped tests for 4.11
-cat <<EOF >>"${SKIP_TESTS_FILE}"
-# <feature> <test name>
-
-EOF
-}
-
-
-function create_tests_temp_skip_list_12 {
-# List of temporarly skipped tests for 4.12
-cat <<EOF >>"${SKIP_TESTS_FILE}"
-# <feature> <test name>
-
-EOF
-}
-
-function create_tests_temp_skip_list_13 {
-# List of temporarly skipped tests for 4.13
-cat <<EOF >>"${SKIP_TESTS_FILE}"
-# <feature> <test name>
-
-EOF
-}
-
-function create_tests_temp_skip_list_14 {
-# List of temporarly skipped tests for 4.14
-cat <<EOF >>"${SKIP_TESTS_FILE}"
-# <feature> <test name>
-
-# SKIPTEST
-# bz### Link to OCPBUGS in JIRA
-# TESTNAME
-ptp "Some PTP test to skip for 4.14 release only"
-
-EOF
+    # Sleep for the specified interval before retrying
+    sleep ${interval}
+  done
 }
 
-# Whether node is Baremetal or not (virtual for example)
-# We are interested in Baremetal nodes only for testing
-function is_bm_node {
-    node=$1
-
-    machine=$(oc get "${node}" -o json | jq '.metadata.annotations' | grep "machine.openshift.io/machine" | cut -d ":" -f2 | tr -d '", ')
-    machine_ns=$(echo "${machine}" | cut -d "/" -f1)
-    machine_name=$(echo "${machine}" | cut -d "/" -f2)
-    bmh=$(oc get machine -n "${machine_ns}" "${machine_name}" -o json | jq '.metadata.annotations' | grep "metal3.io/BareMetalHost" | cut -d ":" -f2 | tr -d '", ')
-    bmh_ns=$(echo "${bmh}" | cut -d "/" -f1)
-    bmh_name=$(echo "${bmh}" | cut -d "/" -f2)
-    manufacturer=$(oc get bmh -n "${bmh_ns}" "${bmh_name}" -o json | jq '.status.hardware.systemVendor.manufacturer')
-    # if the system manufacturer is not Red Hat, that's a BM node
-    if [[ "${manufacturer}" != *"Red Hat"* ]]; then
-        return 0
+forceDeleteNamespace() {
+    oc delete --wait=false namespace openshift-ptp
+    oc get namespace openshift-ptp -o json >${SHARED_DIR}/tmp.json
+    if [ ! $? -ne 0 ]
+    then 
+    sed -i '/"kubernetes"/d' ${SHARED_DIR}/tmp.json
+    oc proxy&
+    sleep 2
+    curl -k -H "Content-Type: application/json" -X PUT --data-binary @${SHARED_DIR}/tmp.json http://127.0.0.1:8001/api/v1/namespaces/openshift-ptp/finalize
+    kill %1
     fi
-    return 1
 }
 
-function get_skip_tests {
-
-    skip_list=""
-    if [ -f "${SKIP_TESTS_FILE}" ]; then
-        rm -f feature_skip_list.txt
-        grep --text -E "^[^#]" "${SKIP_TESTS_FILE}" > feature_skip_list.txt
-        skip_list=""
-        while read line;
-        do
-            test=$(echo "${line}" | cut -d " " -f2- | tr " " .)
-            if [ ! -z "${test}" ]; then
-                if [ "${skip_list}" == "" ]; then
-                    skip_list="${test}"
-                else
-                    skip_list="${skip_list} ${test}"
-                fi
-            fi
-        done < feature_skip_list.txt
-    fi
-
-    echo "${skip_list}"
-}
+set -o nounset
+set -o errexit
+set -o pipefail
 
 source $SHARED_DIR/main.env
```

Comment:
This should be removed, it was created in steps that don't exist in PTP job.
(from here: https://github.com/openshift/release/blob/master/ci-operator/step-registry/telco5g/jobs-router/telco5g-jobs-router-commands.sh)

Reply:
fixed